cmd/server.go

```diff
@@ -10,50 +10,58 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var onFlag, offFlag bool
+
 var serverCmd = &cobra.Command{
 	Use:   "server",
 	Short: "Print server list",
 	Long:  `Print all server information`,
 	Run: func(cmd *cobra.Command, args []string) {
 		ctx := context.Background()
-
 		out := new(bytes.Buffer)
 		servers, err := client.GetServerList(ctx)
```

Comment:
you should use `client.GetServer(ctx, server_uuid)`. As server UUID is given from cmd argument.

Reply:
Yep